### DISCUSSION TITLE
Copilot Labs not working

### DISCUSSION THREAD

**Discussion by diegotecbr** [community/community#52539] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Estou há alguns dias usando o Copilot, porém ao tentar utilizar o Copilot labs, estou recebendo o seguinte erro: Something went wrong checking access: read ECONNRESET

**Comment by Akash1134** [comment]
Hi @diegotecbr ,

Are you using a proxy setup to work with Copilot Labs? If so, it would be worth checking out [this discussion](https://github.com/orgs/community/discussions/29127) for the setup that helped other users. Let us know how it goes.
